src/rank_llm/rerank/rankllm.py

```diff
@@ -331,6 +334,8 @@ def covert_doc_to_prompt_content(self, doc: Dict[str, Any], max_length: int) ->
             content = doc["segment"]
         elif "contents" in doc:
             content = doc["contents"]
+        elif "body" in doc:
+            content = doc["body"]
```

Comment:
Why did you reintroduce this?

Reply:
because the doc object in Anserini's API response only has a body section rather than text,segment, or contents